scripts/avg_rule_profile.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+#
+if [ $# -ne 2 ]; then
+    echo "Not enough arguments arguments provided, you need to provide 3 arguments:";
+    echo "All args are boolean (0 or 1)";
+    echo "";
+    echo "If 1st Argument == 1: Create File with average profiles for a state and alphabet";
+    echo "bash avg_rule_profile.sh 1 0";
+    echo "If 2nd Argument == 1: Plots average of File";
+    echo "bash avg_rule_profile.sh 0 1";
+    echo "";
+    exit 1;
+fi
+
+
+# ==============================================================================
+# Variables
+# ==============================================================================
+FILE=$1;
+PLOT=$2;
+# Default Values
+NUMBERITERATION=1001;
+K=2;
+SavePathText="../resultText";
+SavePathPlot="../resultPlots";
+
+# ==============================================================================
+# Average of Profiles
+# ==============================================================================
+
+function getprofile {
+    while read line; do
+        #get k
+        #get Machine
+        TM=`echo $line | awk '{ print $1;}'`;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use $(...) notation instead of legacy backticked `...`.](https://app.codacy.com/app/jorgeMFS/TMCompression/pullRequest?prid=4701004)